### ISSUE TITLE
[Bug]: Cannot retrieve documents from Chroma / Chromadb running in client server mode 

### ISSUE THREAD

**Issue by concurrent-ai** [run-llama/llama_index#11420]
### Bug Description

The following code results in an error.  Is there a fix or workaround for this?

```
File "/home/dev/miniconda3/envs/infinstor/lib/python3.11/site-packages/chromadb/api/fastapi.py", line 577, in _query
     data=json.dumps(
          ^^^^^^^^^^^
TypeError: Type is not JSON serializable: numpy.float64
```

```python
#!/usr/bin/env python

###############
#  Code below from https://docs.llamaindex.ai/en/stable/examples/vector_stores/ChromaIndexDemo.html 
################

# import
from llama_index import VectorStoreIndex, SimpleDirectoryReader
from llama_index.vector_stores.chroma import ChromaVectorStore
from llama_index import StorageContext
from llama_index.embeddings.huggingface import HuggingFaceEmbedding
import llama_index
import chromadb
import traceback_with_variables

traceback_with_variables.global_print_exc()

# define embedding function
embed_model = HuggingFaceEmbedding(model_name="BAAI/bge-small-en-v1.5") #"BAAI/bge-base-en-v1.5")

# load documents
documents = SimpleDirectoryReader("./data/paul_graham/").load_data()

# create the chroma client and add our data
import chromadb

remote_db = chromadb.HttpClient(host="127.0.0.1", port=8000)
chroma_collection = remote_db.get_or_create_collection("quickstart")
vector_store = ChromaVectorStore(chroma_collection=chroma_collection)
storage_context = StorageContext.from_defaults(vector_store=vector_store)

# Error: To disable the LLM entirely, set llm=None.
index = VectorStoreIndex.from_documents(
    documents, storage_context=storage_context, embed_model=embed_model, service_context=llama_index.ServiceContext.from_defaults(llm=None, embed_model=embed_model)
)

# Query Data from the Chroma Docker index
query_engine = index.as_query_engine()
# Error below is seen:
#   File "/home/dev/miniconda3/envs/infinstor/lib/python3.11/site-packages/chromadb/api/fastapi.py", line 577, in _query
#     data=json.dumps(
#          ^^^^^^^^^^^
# TypeError: Type is not JSON serializable: numpy.float64
response = query_engine.query("What did the author do growing up?")
print(f"<b>{response}</b>")
```

### Version

0.9.46

### Steps to Reproduce

* start chroma in docker using the command: docker run --publish 8000:8000 chromadb/chroma
* Run the code in the bug description
* The error shown will be seen

### Relevant Logs/Tracbacks

```shell
./chroma_example_from_llama_index.py 
LLM is explicitly disabled. Using MockLLM.
Traceback with variables (most recent call last):
  File "./chroma_example_from_llama_index.py", line 44, in <module>
    response = query_engine.query("What did the author do growing up?")
      __name__ = '__main__'
      __doc__ = None
      __package__ = None
      __loader__ = <_frozen_importlib_external.SourceFileLoader object at 0x7f5b76d02950>
      __spec__ = None
      __annotations__ = {}
      __builtins__ = <module 'builtins' (built-in)>
      __file__ = './chroma_example_from_llama_index.py'
      __cached__ = None
      VectorStoreIndex = <class 'llama_index.indices.vector_store.base.VectorStoreIndex'>
      SimpleDirectoryReader = <class 'llama_index.readers.file.base.SimpleDirectoryReader'>
      ChromaVectorStore = <class 'llama_index.vector_stores.chroma.ChromaVectorStore'>
      StorageContext = <class 'llama_index.storage.storage_context.StorageContext'>
      HuggingFaceEmbedding = <class 'llama_index.embeddings.huggingface.HuggingFaceEmbedding'>
      llama_index = <module 'llama_index' from 'lib/python3.11/site-packages/llama_index/__init__.py'>
      chromadb = <module 'chromadb' from 'lib/python3.11/site-packages/chromadb/__init__.py'>
      traceback_with_variables = <module 'traceback_with_variables' from 'lib/python3.11/site-packages/traceback_with_variables/__init__.py'>
      embed_model = HuggingFaceEmbedding(model_name='BAAI/bge-small-en-v1.5', embed_batch_size=10, callback_manager=<llama_index.callbacks.base.CallbackManager object at 0x7f5a959f3390>, tokenizer_name='BAAI/bge-small-en-v1.5', max_length=512, pooling=<Pooling.CLS: 'cls'>, normalize=True, query_instruction=None, text_instruction=None, cache_folder=None)
      documents = [Document(id_='0af0cf4e-e391-462d-a62a-f7418170b896', embedding=None, metadata={'file_path': 'data/paul_graham/paul_graham_essay.txt', 'file_name': 'paul_graham_essay.txt', 'file_type': 'text/plain', 'file_size': 75042, 'creation_date': '2024-02-25', 'last_modified_date': '2024-02-25', 'last_accessed_date': '2024-02-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='\n\nWhat I Worked On\n\nFebruary 2021\n\nBefore college the two main things I worked on, outside of school, were writing and programming. I didn\'t write essays. I wrote what beginning writers were supposed to write then, and probably still are: short stories. My stories were awful. They had hardly any plot, just characters with strong feelings, which I imagined made them deep.\n\nThe firs...
      remote_db = <chromadb.api.client.Client object at 0x7f5a95aacf50>
      chroma_collection = Collection(name=quickstart)
      vector_store = ChromaVectorStore(stores_text=True, is_embedding_query=True, flat_metadata=True, collection_name=None, host=None, port=None, ssl=False, headers=None, persist_dir=None, collection_kwargs={})
      storage_context = StorageContext(docstore=<llama_index.storage.docstore.simple_docstore.SimpleDocumentStore object at 0x7f5a901e2a10>, index_store=<llama_index.storage.index_store.simple_index_store.SimpleIndexStore object at 0x7f5a959d6e50>, vector_stores={'default': ChromaVectorStore(stores_text=True, is_embedding_query=True, flat_metadata=True, collection_name=None, host=None, port=None, ssl=False, headers=None, persist_dir=None, collection_kwargs={}), 'image': <llama_index.vector_stores.simple.SimpleVectorStore object at 0x7f5a95928f50>}, graph_store=<llama_index.graph_stores.simple.SimpleGraphStore object at 0x7f5a959f32d0>)
      index = <llama_index.indices.vector_store.base.VectorStoreIndex object at 0x7f5a8d007ad0>
      query_engine = <llama_index.query_engine.retriever_query_engine.RetrieverQueryEngine object at 0x7f5a8dadadd0>
  File "lib/python3.11/site-packages/llama_index/core/base_query_engine.py", line 40, in query
    return self._query(str_or_query_bundle)
      self = <llama_index.query_engine.retriever_query_engine.RetrieverQueryEngine object at 0x7f5a8dadadd0>
      str_or_query_bundle = QueryBundle(query_str='What did the author do growing up?', image_path=None, custom_embedding_strs=None, embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.0370...
  File "lib/python3.11/site-packages/llama_index/query_engine/retriever_query_engine.py", line 171, in _query
    nodes = self.retrieve(query_bundle)
      self = <llama_index.query_engine.retriever_query_engine.RetrieverQueryEngine object at 0x7f5a8dadadd0>
      query_bundle = QueryBundle(query_str='What did the author do growing up?', image_path=None, custom_embedding_strs=None, embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.0370...
      query_event = <llama_index.callbacks.base.EventContext object at 0x7f5a8ce020d0>
  File "lib/python3.11/site-packages/llama_index/query_engine/retriever_query_engine.py", line 127, in retrieve
    nodes = self._retriever.retrieve(query_bundle)
      self = <llama_index.query_engine.retriever_query_engine.RetrieverQueryEngine object at 0x7f5a8dadadd0>
      query_bundle = QueryBundle(query_str='What did the author do growing up?', image_path=None, custom_embedding_strs=None, embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.0370...
  File "lib/python3.11/site-packages/llama_index/core/base_retriever.py", line 224, in retrieve
    nodes = self._retrieve(query_bundle)
      self = <llama_index.indices.vector_store.retrievers.retriever.VectorIndexRetriever object at 0x7f5a8dafeed0>
      str_or_query_bundle = QueryBundle(query_str='What did the author do growing up?', image_path=None, custom_embedding_strs=None, embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.0370...
      query_bundle = QueryBundle(query_str='What did the author do growing up?', image_path=None, custom_embedding_strs=None, embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.0370...
      retrieve_event = <llama_index.callbacks.base.EventContext object at 0x7f5a8d01bd50>
  File "lib/python3.11/site-packages/llama_index/indices/vector_store/retrievers/retriever.py", line 92, in _retrieve
    return self._get_nodes_with_embeddings(query_bundle)
      self = <llama_index.indices.vector_store.retrievers.retriever.VectorIndexRetriever object at 0x7f5a8dafeed0>
      query_bundle = QueryBundle(query_str='What did the author do growing up?', image_path=None, custom_embedding_strs=None, embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.0370...
  File "lib/python3.11/site-packages/llama_index/indices/vector_store/retrievers/retriever.py", line 168, in _get_nodes_with_embeddings
    query_result = self._vector_store.query(query, **self._kwargs)
      self = <llama_index.indices.vector_store.retrievers.retriever.VectorIndexRetriever object at 0x7f5a8dafeed0>
      query_bundle_with_embeddings = QueryBundle(query_str='What did the author do growing up?', image_path=None, custom_embedding_strs=None, embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.0370...
      query = VectorStoreQuery(query_embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.03702234476804733, -0.004152943380177021, -0.05241239070892334, 0.0007901482167653739,...
  File "lib/python3.11/site-packages/llama_index/vector_stores/chroma.py", line 300, in query
    results = self._collection.query(
      self = ChromaVectorStore(stores_text=True, is_embedding_query=True, flat_metadata=True, collection_name=None, host=None, port=None, ssl=False, headers=None, persist_dir=None, collection_kwargs={})
      query = VectorStoreQuery(query_embedding=[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.03702234476804733, -0.004152943380177021, -0.05241239070892334, 0.0007901482167653739,...
      kwargs = {}
      where = {}
  File "lib/python3.11/site-packages/chromadb/api/models/Collection.py", line 345, in query
    query_results = self._client._query(
      self = Collection(name=quickstart)
      query_embeddings = [0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.03702234476804733, -0.004152943380177021, -0.05241239070892334, 0.0007901482167653739, 0.026182137429714203, 0.00773368...
      query_texts = None
      query_images = None
      query_uris = None
      n_results = 2
      where = {}
      where_document = None
      include = ['metadatas', 'documents', 'distances']
      valid_where = {}
      valid_where_document = {}
      valid_query_embeddings = [[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.03702234476804733, -0.004152943380177021, -0.05241239070892334, 0.0007901482167653739, 0.026182137429714203, 0.0077336...
      valid_query_texts = None
      valid_query_images = None
      valid_query_uris = None
      valid_include = ['metadatas', 'documents', 'distances']
      valid_n_results = 2
  File "lib/python3.11/site-packages/chromadb/telemetry/opentelemetry/__init__.py", line 127, in wrapper
    return f(*args, **kwargs)
      args = (<chromadb.api.fastapi.FastAPI object at 0x7f5a959d6790>,)
      kwargs = {'collection_id': UUID('c5611752-2efe-4f8e-b749-542c23a85dc5'), 'query_embeddings': [[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.03702234476804733, -0.004152943380...
      attributes = None
      f = <function FastAPI._query at 0x7f5a901e53a0>
      trace_granularity = <OpenTelemetryGranularity.ALL: 'all'>
      trace_name = 'FastAPI._query'
  File "lib/python3.11/site-packages/chromadb/api/fastapi.py", line 577, in _query
    data=json.dumps(
      self = <chromadb.api.fastapi.FastAPI object at 0x7f5a959d6790>
      collection_id = UUID('c5611752-2efe-4f8e-b749-542c23a85dc5')
      query_embeddings = [[0.009083932265639305, 0.045616503804922104, 0.04721105471253395, 0.0021174883004277945, -0.024913949891924858, -0.03000902198255062, -0.028638124465942383, 0.006340742111206055, -0.007490871474146843, 0.040900230407714844, -0.01630522310733795, 0.015754150226712227, -0.02374003455042839, 0.014889922924339771, 0.029421977698802948, 0.03340929001569748, 0.015343260951340199, 0.029110386967658997, -0.06259063631296158, -0.038029082119464874, -0.05742312967777252, 0.0006062944885343313, 0.031412359327077866, -0.07837338000535965, 0.08854351937770844, 0.03195532038807869, -0.022621657699346542, 0.027211694046854973, -0.07398305088281631, -0.09235735982656479, 0.023819297552108765, 0.03896259143948555, 0.031853239983320236, 0.014170384034514427, 0.004976786673069, 0.07296357303857803, 0.01744353398680687, 0.025804976001381874, 0.003496797289699316, 0.0038374830037355423, 0.03702234476804733, -0.004152943380177021, -0.05241239070892334, 0.0007901482167653739, 0.026182137429714203, 0.0077336...
      n_results = 2
      where = {}
      where_document = {}
      include = ['metadatas', 'documents', 'distances']
builtins.TypeError: Type is not JSON serializable: numpy.float64
```

**Comment by concurrent-ai** [comment]
@dosu-bot

I cannot make the change you've suggested above since the computing of the embedding happens inside llama-index, when the following code is invoked, and it isn't done in my code.

```python
response = query_engine.query("What did the author do growing up?")
```

I can compute the embedding of the query, and create a QueryBundle with the embedding as you've suggested, but that is not the ideal way to work with the llama-index query_engine.

Yes, if the maintainers of chromadb integration can take a look, they can help resolve this.  Looks like @stmtk1 or @poxrud or @Disiok or @jerryjliu are the maintainers of this integration?

**Comment by software-artisan** [comment]
@dosubot

> _🤖_
> 
> Thank you for providing detailed information about the issue you're encountering. It seems like the problem is related to the serialization of numpy.float64 types in the LlamaIndex package. To better assist you, I need a bit more information:
> 
> 1. Could you please provide the full traceback of the error you're encountering? This will help me understand the context in which the error is occurring.
full Traceback is already attached to the error
> 2. Are you using any specific configuration for the ChromaVectorStore or the query engine?
Chroma running in client server mode is the configuration.
> 3. Have you made any modifications to the LlamaIndex package or are you using it as is?
No
> 4. Could you also share the version of numpy you're using?
1.26.3
> 
> This additional information will help me understand the issue more clearly and provide a more accurate solution.
> 
> #### About Dosu

**Comment by concurrent-ai** [comment]
@dosubot

> _🤖_
> 
> Thank you for providing detailed information about the issue you're encountering. It seems like the problem is related to the serialization of numpy.float64 types in the LlamaIndex package. To better assist you, I need a bit more information:
> 
> 1. Could you please provide the full traceback of the error you're encountering? This will help me understand the context in which the error is occurring.

full Traceback is already attached to the error

> 2. Are you using any specific configuration for the ChromaVectorStore or the query engine?

Chroma running in client server mode is the configuration.

> 3. Have you made any modifications to the LlamaIndex package or are you using it as is?

No

> 4. Could you also share the version of numpy you're using?

1.26.3

> 
> This additional information will help me understand the issue more clearly and provide a more accurate solution.
> 
> #### About Dosu

**Comment by software-artisan** [comment]
@dosubot

It is not the add() that has problems.  It is the retrieve() that has problems.

**Comment by poxrud** [comment]
This PR https://github.com/run-llama/llama_index/pull/11458 by @TonyBotongChu  fixes this.

**Comment by emmepra** [comment]
Anyone solved this somehow?

**Comment by TonyBotongChu** [comment]
> Anyone solved this somehow?

Yes it's fixed in 0.10.14 release.
